interpreter/cling/lib/Interpreter/IncrementalCUDADeviceCompiler.cpp

```diff
@@ -159,6 +157,11 @@ namespace cling {
         additionalPtxOpt.push_back(s);
     }
 
+    // add manual path to CUDA SDK
```

Comment:
I don't understand "manual path"?

Reply:
Yes, `manual path` makes no sense. I think `use custom CUDA SDK path` is better.